docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -3,34 +3,97 @@ title: Tic-Tac-Toe
+
+## What you need
+
+Before getting started, make sure you have:
+
+- [Installed the latest version of Sui](../getting-started/sui-install.mdx).
+- Read the basics of [shared versus owned objects](../sui-101/shared-owned.mdx).
+
+## Directory structure
+
+To begin, create a new folder on your system titled `tic-tac-toe` that holds all your files.
+
+In this folder, create the following subfolders: 
+- `move` to hold the Move code for the game board.
+  - `sources` to hold the Move source files.
+
+<details>
+  <summary>
+    Add `Move.toml` to `tic-tac-toe/move/`
+  </summary>
+  ```toml
+  [package]
+  name = "tic_tac_toe"
+  edition = "2024.beta"
+
+  [dependencies]
+  Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/testnet" }
+
+  [addresses]
+  tic_tac_toe = "0x0"
+  ```
+</details>
+
+:::checkpoint
+
+- You have the latest version of Sui installed. If you run `sui --version` in your terminal or console, it responds with the currently installed version.
+- You have a directory to place the files you create in. 
+- You have created a `Move.toml` file in the `tic-tac-toe/move/` directory.
+
+:::
+
+## owned.move {#owned.move}
+
+Create a new file in `tic-tac-toe/move/sources` titled `owned.move`. This file will contain the Move code for the game board in the centralized (and multisig) version of tic-tac-toe.
+
+In this first example of tic-tac-toe, the `Game` object, including the game board, is controlled by a game admin.
+
+{@inject: examples/tic-tac-toe/move/sources/owned.move#struct=Game noComments}
+
+Ignore the `admin` field for now, as it is only relevant for the multisig approach.
 
 Games are created with the `new` function:
 
 {@inject: examples/tic-tac-toe/move/sources/owned.move#fun=new noComments}
 
-Some things to note:
-- The game is created and returned by this function, it is up to its creator to send it to the game admin to own.
-- There is an `admin` field, we can ignore this for now, as it is only relevant for the multisig approach.
-- The first player is sent a `TurnCap` which gives them permission to take the next turn, they consume it to make their `Mark`, and the admin mints and sends a new `TurnCap` to the next player if the game has not ended.
+Some things to notes: 
+- `MARK__` is a constant that represents an empty cell on the game board. `MARK_X` and `MARK_O` represent the two players' markers.
+- The first player is sent a `TurnCap` which gives them permission to take the next turn.
+- The `Game` object is created and returned by this function, it is up to its creator to send it to the game admin to own.
+
+Because the players don’t own the game board, they cannot directly mutate it. Instead, they indicate their move by creating a `Mark` object with their intended placement and send it to the game object using transfer to object:
```

Comment:
```suggestion
Because the players don’t own the game board object, they cannot directly mutate it. Instead, they indicate their move by creating a `Mark` object with their intended placement and send it to the game object using transfer to object:
```
nittiest of nits